classes/SectionView/SeneddView.php

```diff
@@ -98,11 +98,43 @@ protected function display_front_senedd() {
         $data['debates'] = array( 'recent' => $recent);
 
         $data['regional'] = $this->getMSList();
+        $data['search_box'] = $this->getSearchBox($data);
         $data['template'] = 'senedd/index';
 
         return $data;
     }
 
+    protected function getSearchBox(array $data): \MySociety\TheyWorkForYou\Search\SearchBox{
+        $search_box = new \MySociety\TheyWorkForYou\Search\SearchBox();
+        $search_box->homepage_panel_class = "panel--homepage--senedd";
+        if (LANGUAGE == 'cy') {
+            $search_box->homepage_subhead = "Senedd";
+        } else {
+            $search_box->homepage_subhead = "Senedd / Welsh Parliament";
+        }
+        $search_box->homepage_desc = "";
+        $search_box->search_section = "senedd";
+        $search_box->quick_links = [];
+        if (count($data["regional"])) {
+            // get all unique constituencies
+            $constituencies = array();
+            foreach ($data["regional"] as $member) {
+                $constituencies[$member["constituency"]] = 1;
+            }
+            $constituencies = array_keys($constituencies);
+            if (LANGUAGE == 'cy'){
```

Comment:
Feels like this could be done with a string and an actual translation call to keep the translations in the translation file, but no biggie.